packages/frontend/src/helpers/tests.jsx

```diff
@@ -32,8 +33,22 @@ const setupTests = (
   return { screen, user };
 };
 
-const createSetupTestsForRoute = (route) => {
+const createSetupTestsForRoute = route => {
   return (component, options) => setupTests(component, { ...options, route });
 };
 
-export { setupTests, createSetupTestsForRoute };
+const Providers = ({ children }) => (
+  <BrowserRouter>
+    <FormProvider>
+      <PhotosProvider>
+        <GameProvider>{children} </GameProvider>
+      </PhotosProvider>
+    </FormProvider>
+  </BrowserRouter>
+);

```

Comment:
I think this could be used within the setupTests function, so that the way the providers are rendered are the same throughout all the tests.

```javascript
const Providers = ({ children, state = baseState }) => (
  <BrowserRouter>
    <FormProvider providedState={{ ...state.form }}>
      <PhotosProvider providedState={{ ...state.photos }}>
        <GameProvider providedState={{ ...state.game }}>
           {children}
        </GameProvider>
      </PhotosProvider>
    </FormProvider>
  </BrowserRouter>
);
// now the Providers component can verify prop types without needing to rewrite them in each test nor ignoring them with eslint
Providers.propTypes = PropTypes.shape({
  children: PropTypes.node, // this might need to change for rendering a hook?
  state: PropTypes.object,
})

const setupTests = (
  Component,
  { props, state = baseState, route = '/' } = {}
) => {
  // push the route into the "browser" history
  window.history.pushState({}, 'Test', route);
  const user = userEvent.setup({ delay: null });
  render(
    <Providers state={state}>
       <Component {...props}/>
    </Providers>
  );
```